app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
Also, like I mentioned earlier, `5.times { @paper.authors.build }` actually evaluates to `5`.

```
irb(main):002:0> 1 + 5.times {print 'hey'}
heyheyheyheyhey=> 6
```